### PR TITLE
My changes adding pushups

### PR DESCRIPTION
**Pull request by mdaz78**

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: english/light-fire/eating/rest/pushups.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Do some push ups
```
Comment (casey-collab) [inline english/light-fire/eating/rest/pushups.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**